tools/util_test.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+from util import pattern_match
+
+
+def pattern_match_test():
+    print "Testing util.pattern_match()..."
+    fixtures_expected = [
+        "foobarbaz", "[WILDCARD]", "foobar", "foo[WILDCARD]baz",
+        "foo[WILDCARD]baz", "foo[WILDCARD]baz[WILDCARD]qux", "foo[WILDCARD]",
+        "foo[WILDCARD]baz[WILDCARD]"
+    ]
+    fixtures_actual = [
+        "foobarbaz", "foobarbaz", "foobarbaz", "foobarbaz", "foobazbar",
+        "foobarbazqatqux", "foobar", "foobarbazqat"
+    ]
+    fixtures_results = [True, True, False, True, False, True, True, True]
+
+    # Iterate through the fixture lists, testing each one
+    for i in range(0, len(fixtures_expected)):
```

Comment:
I think this would be more readable 
```
fixtures = [ ( "foobarbaz", "foobarbaz", True),
             ("[WILDCARD]", "foobarbaz", True), 
             # ... 
           ]
for (string, pattern, expected) in fixtures:
  actual = pattern_match(string, pattern)
  assert expected == actual
```